.github/templates/windows_ci_workflow.yml.j2

```diff
@@ -37,12 +40,12 @@ env:
   VS_VERSION: "16.8.6"
   VC_YEAR: "2019"
   ALPINE_IMAGE: "308535385114.dkr.ecr.us-east-1.amazonaws.com/tool/alpine"
+  no_proxy: !{{ squid_no_proxy }}
```

Comment:
Any reason we don't set `http_proxy` at the workflow level too?

Reply:
I'm trying to simulate what linux CI is doing is to limit the proxy to only `build` and `test` jobs, linux CI runs those in docker, windows CI runs in plain steps, thus the differences.